client/ayon_silhouette/plugins/create/create_workfile.py

```diff
@@ -0,0 +1,128 @@
+from ayon_core.pipeline import CreatedInstance, AutoCreator, AYON_INSTANCE_ID
+
+from ayon_silhouette.api import lib
+
+import fx
+
+
+class CreateWorkfile(AutoCreator):
+    """Workfile auto-creator."""
+    identifier = "io.ayon.creators.silhouette.workfile"
+    label = "Workfile"
+    product_type = "workfile"
+    icon = "fa5.file"
+
+    project_property_name = "AYON_workfile"
+    default_variant = "Main"
+
+    def create(self):
+        """Create workfile instances."""
+        if not fx.activeProject():
+            return
+
+        workfile_instance = next(
+            (
+                instance for instance in self.create_context.instances
+                if instance.creator_identifier == self.identifier
+            ),
+            None,
+        )
+
+        project_entity = self.create_context.get_current_project_entity()
+        project_name = project_entity["name"]
+        folder_entity = self.create_context.get_current_folder_entity()
+        folder_path = folder_entity["path"]
+        task_entity = self.create_context.get_current_task_entity()
+        task_name = task_entity["name"]
+        host_name = self.create_context.host_name
+
+        variant = self.default_variant
+        if not workfile_instance:
+            product_name = self.get_product_name(
+                project_name,
+                folder_entity,
+                task_entity,
+                variant,
+                host_name,
+                project_entity=project_entity
+            )
+            data = {
+                "folderPath": folder_path,
+                "task": task_name,
+                "variant": variant,
+            }
+
+            # Enforce forward compatibility to avoid the instance to default
+            # to the legacy `AVALON_INSTANCE_ID`
+            data["id"] = AYON_INSTANCE_ID
```

Comment:
@iLLiCiTiT As soon as I remove this - then the imprinted data will have `id` value `pyblish.avalon.instance`.

Reply:
The source of that is here: https://github.com/ynput/ayon-core/blob/2cf62f0bb455c401897f85f641774e70770ca1fa/client/ayon_core/pipeline/create/structures.py#L493-L495

So to ONLY have AYON_INSTANCE_ID in this new integration I need to explicitly make sure to currently force the newer values.

Reply:
> So to ONLY have AYON_INSTANCE_ID in this new integration I need to explicitly make sure to currently force the newer values.

Will create PR in ayon-core (hopefully nothing breaks?).